### PR TITLE
Bug 1719315 - Add support for the URL metric type

### PR DESCRIPTION
**Pull request by brizental**
### Pull Request checklist ###

- [x] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry to `CHANGELOG.md` or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,13 @@
 
 ## Unreleased
 
-- New lint: Check for redundant words in ping names ([#355](https://github.com/mozilla/glean_parser/pull/355))
+- Add support for URL metric type ([#361](https://github.com/mozilla/glean_parser/pull/361))
```
Comment (badboy) [inline CHANGELOG.md:5]: You will need to rebase, so we don't lose the other change message here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,15 +2,13 @@
...
 
 ## 3.6.0 (2021-06-11)
 
 - Add a command `data-review` to generate a skeleton Data Review Request for all metrics matching a supplied bug number. ([bug 1704541](https://bugzilla.mozilla.org/show_bug.cgi?id=1704541))
 - Enable custom distribution outside of GeckoView (`gecko_datapoint` becomes optional)
-
 ## 3.5.0 (2021-06-03)
-
```
Comment (badboy) [inline CHANGELOG.md:13]: Mistakenly removed newlines here?

**Review by badboy — COMMENTED** [review]
This isn't adding anything but the most basic parsing.
I assume because it's that easy it won't require any template changes, correct?

File: CHANGELOG.md (hunk 1 above)
Comment (brizental) [reply CHANGELOG.md:5]: I did rebase... just did it wrong heh. I'll fix it.

**Comment by brizental** [comment]
> This isn't adding anything but the most basic parsing.
> I assume because it's that easy it won't require any template changes, correct?

Exactly. This is how the generated code looks like:

```js
import UrlMetricType from "@mozilla/glean/webext/private/metrics/url";
/**
 * ...
 *
 * Generated from `telemetry.test`.
 */
export const test = new UrlMetricType({
    category: "telemetry",
    name: "test",
    sendInPings: ["metrics"],
    lifetime: "user",
    disabled: false,
});
```

**Comment by badboy** [comment]
> > This isn't adding anything but the most basic parsing.
> > I assume because it's that easy it won't require any template changes, correct?
> 
> Exactly. This is how the generated code looks like:
> 
> ```js
> import UrlMetricType from "@mozilla/glean/webext/private/metrics/url";
> /**
>  * ...
>  *
>  * Generated from `telemetry.test`.
>  */
> export const test = new UrlMetricType({
>     category: "telemetry",
>     name: "test",
>     sendInPings: ["metrics"],
>     lifetime: "user",
>     disabled: false,
> });
> ```

It's like we planned for this!

**Review by badboy — APPROVED** [review]

**Pull request merged**